test/models/group_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class GroupTest < ActiveSupport::TestCase
```

Comment:
@nikitr Try not to include auto-generated files. Remove if nothing useful is added